Automattic/newspack-popups
# Update/refine cli commands

### All Submissions:






### Changes proposed in this Pull Request:



Refines the CLI commands giving more options, localizing it and also fixing some issues in the importer / exporter

### How to test the changes in this Pull Request:

Please use this Pull request to test the full extent of the importer / exporter feature

1. Create a (or using an existing site) with many prompts, campaigns and segments
2. Make sure you have prompts in different states (publish, draft, etc)
3. Make sure to use tags and categories in all possible fields (tags and categories in prompts, excluded_tags and excluded_categories in prompts and favorite_categories in segments)
4. Run the exporter CLI command
5. does the doc and the messaging look good?
6. Now fire a brand new site
7. Create at least one of the categories that existed in the site you did the export (just one!)
8. Create a new category with a different name - that didn't exist in the site you did the export
9. Look at the importer docs `wp newspack-popups export --help`
10. can you understand it? can you improve it? any typos?
11. Run the importer
12. You are going to be prompted to what to do about the missing categories and tags
13. Try different options for each one. See how the interaction feels
14. Choose at least one dfifferent option once
15. When you choose to map to an existing category, choose the category you created in step 8
16. You should NOT be asked about the category you created in step 7
17. After the import is complete, go to the admin and check things were created correctly
18. Check all tags and categories in all fields
19. on a new site, again, run the importer and test the `--skip-terms` and `--create-terms` flags.

### Other information:
